### PR TITLE
fix: controller namespace

### LINKED ISSUE #116: Controller namespace causing some errors
I created API controller with same kind of syntax than in Laracast example and there are couple of namespacing errors.

draft.yaml used:

```yaml
models:
  Contact:
    name: string:100
    email: string:255
    phone: nullable string:50
    message: text
    replied_at: nullable timestamp
    reply: nullable text
controllers:
  Api\Contact:
    store:
      validate: name, email, phone, message
      save: contact
      send: ContactNotification to:user with:contact
      fire: NewContact with:contact
      respond: 204
```

Errors:

* In controller `model` is referenced `use App\Api\Contact;` but it should be ``use App\Contact;``
* In Feature test `request` is referenced `\App\Http\Requests\ContactStoreRequest::class` but it should be `\App\Http\Requests\Api\ContactStoreRequest::class`
* `routes/web.php` `ContactController` => `Api\ContactController`

### PR DESCRIPTION
**Pull request by nathane**
close #116

### CODE REVIEW

File: src/Generators/TestGenerator.php [hunk 1]
```diff
@@ -155,7 +155,13 @@ protected function buildTestCases(Controller $controller)
                 } elseif ($statement instanceof ValidateStatement) {
                     $this->addTestAssertionsTrait($controller);
 
-                    $class = $controller->name() . Str::studly($name) . 'Request';
+                    $class = $this->buildFormRequestName($controller, $name);
+//                    dd([
```
Comment (jasonmccreary) [inline src/Generators/TestGenerator.php:159]: Please remove _dead code_.

File: tests/fixtures/routes/respond-statements.php [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (jasonmccreary) [inline tests/fixtures/routes/respond-statements.php:1]: Why are there two blank lines?

Comment (nathane) [reply tests/fixtures/routes/respond-statements.php:1]: https://github.com/laravel-shift/blueprint/blob/db562f297d366e02124a9cc9b6746116aab5faba/src/Generators/RouteGenerator.php#L28

Comment (jasonmccreary) [reply tests/fixtures/routes/respond-statements.php:1]: Hahaha. 😅

**Comment by jasonmccreary** [comment]
Does this close all of the issues mentioned in #116 or just a one?

File: tests/fixtures/routes/respond-statements.php (hunk 2 above)
Comment (nathane) [reply tests/fixtures/routes/respond-statements.php:1]: I've assumed it was the way you wanted it, after looking at. `routes/readme-example.php`

**Comment by nathane** [comment]
this pr closes all issues in #116

File: tests/fixtures/routes/respond-statements.php (hunk 2 above)
Comment (nathane) [reply tests/fixtures/routes/respond-statements.php:1]: do you want me to fix it?

Comment (jasonmccreary) [reply tests/fixtures/routes/respond-statements.php:1]: No. That's okay. It's on my list to review the route generation anyway.

**Pull request merged**